lib/prompts/checkbox.js

```diff
@@ -86,7 +88,8 @@ Prompt.prototype.render = function (error) {
   var bottomContent = '';
 
   if (this.firstRender) {
-    message += '(Press <space> to select)';
+    message += "(Press <space> to select, <a> to select all or none,";
```

Comment:
We need to make this message way shorter now that there's so many keys.

`(Press space to select, a to toggle all, i to inverse selection)`

Maybe with some chalk styling for the key name?